client/src/app/routes/stardust/TransactionPage.tsx

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable max-len */
+import { TRANSACTION_PAYLOAD_TYPE } from "@iota/iota.js-stardust";
+import React, { ReactNode } from "react";
+import { RouteComponentProps } from "react-router-dom";
+import { ServiceFactory } from "../../../factories/serviceFactory";
+import { ClipboardHelper } from "../../../helpers/clipboardHelper";
+import { STARDUST } from "../../../models/config/protocolVersion";
+import { StardustTangleCacheService } from "../../../services/stardust/stardustTangleCacheService";
+import AsyncComponent from "../../components/AsyncComponent";
+import CopyButton from "../../components/CopyButton";
+import Modal from "../../components/Modal";
+import TransactionPayload from "../../components/stardust/TransactionPayload";
+import NetworkContext from "../../context/NetworkContext";
+import mainHeaderMessage from "./../../../assets/modals/address/main-header.json";
+import { TransactionsHelper } from "./../../../helpers/stardust/transactionsHelper";
+import "./TransactionPage.scss";
+import { TransactionPageProps } from "./TransactionPageProps";
+import { TransactionPageState } from "./TransactionPageState";
+
+/**
+ * Component which will show the Transaction page for stardust.
+ */
+class TransactionPage extends AsyncComponent<RouteComponentProps<TransactionPageProps>, TransactionPageState> {
+    /**
+     * The component context type.
+     */
+    public static contextType = NetworkContext;
+
+    /**
+     * API Client for tangle requests.
+     */
+    private readonly _tangleCacheService: StardustTangleCacheService;
+
+    /**
+     * Create a new instance of Transaction.
+     * @param props The props.
+     */
+    constructor(props: RouteComponentProps<TransactionPageProps>) {
+        super(props);
+
+        this._tangleCacheService = ServiceFactory.get<StardustTangleCacheService>(
+            `tangle-cache-${STARDUST}`
+        );
+
+        this.state = {
+            networkId: "",
```

Comment:
No need to set state to empty string. They will be undefined on initialization.

Reply:
You can remove the whole `this.state =`.

Reply:
Done.